docs/reference/setup/configuration.asciidoc

```diff
@@ -71,14 +71,21 @@ path.logs: /var/log/elasticsearch
 
 Environment variables referenced with the `${...}` notation within the
 configuration file will be replaced with the value of the environment
-variable, for instance:
+variable. For example:
 
 [source,yaml]
 --------------------------------------------------
 node.name:    ${HOSTNAME}
 network.host: ${ES_NETWORK_HOST}
 --------------------------------------------------
 
+When specifying values for environment variables, you can only use strings. More complex data structures like lists are not supported. However, settings that accept data structures such as lists also accepted comma-separated strings. For example, you can specify multiple values for the `${HOSTNAME}` variable from the previous example:
+
+[source,yaml]
+----
+export HOSTNAME=“host1”, “host2"
```

Comment:
I don't think this works (in `bash` at least) because it splits the assignment at the space:

```
$ export TEST_ARRAY="host1", "host2"
$ echo $TEST_ARRAY
host1,
```

I also think having two quoted strings is confusing, given that we're trying to construct a single string containing both values separated by a comma. Also we don't permit a space in the string. Suggest this:

```suggestion
export HOSTNAME=“host1,host2"
```